pkg/utils/validator/clusterresourceoverride_test.go

```diff
@@ -373,11 +403,41 @@ func TestValidateClusterResourceOverride(t *testing.T) {
 							Name:    "test-cluster-role",
 						},
 					},
+					Policy: validPolicy,
 				},
 			},
 			croList:    nil,
 			wantErrMsg: nil,
 		},
+		"invalid cluster resource override - fail validateClusterResourceOverrideRuleSelector": {
```

Comment:
add more tests? eg, policy is nil, nil labelSelector, empty terms? empty matchLabels map and empty matchExpressions?